etc/config-template.toml

```diff
@@ -30,8 +30,12 @@ end-point-concurrency = 8
 # capacity = 0 # 0 is unlimited.
 
 [metric]
-# prometheus client report interval, set 0s to disable prometheus.
-report-interval = "5m"
+# prometheus client push interval, set 0s to disable prometheus from pushing.
+interval = "15s"
+# prometheus pushgateway address, leaves it empty will disable prometheus from pushing.
```

Comment:
+# the Prometheus pushgateway address. Leaving it empty stops Prometheus from pushing.